pages/index.js

```diff
@@ -294,7 +326,15 @@ const Home = ({ latestStories }) => {
           </div>
           <div className="col-md">
             <GraphSubtitle>CUMULATIVE CASE COUNT</GraphSubtitle>
-            <GraphNumber noBorder> {cumulativeCase} Cases </GraphNumber>
+            <div class="row">
+              <div class="col-auto">
+                <GraphNumber noBorder> {cumulativeCase} Cases </GraphNumber>
+              </div>
+              <GraphNumberBubble>
+                <div>({Math.round(10000 * cumulativeCase / cumulativeTest) / 100}% positivity rate with</div>
+                <div>{commaNumber(cumulativeTest)} tests administered)</div>
```

Comment:
why are the divs needed here? any necessary spacing can be accomplished with {" "}

Reply:
keeping html as flat as possible helps with accessibility